common/es/src/main/java/io/apiman/common/es/util/DefaultEsClientFactory.java

```diff
@@ -198,12 +195,13 @@ private void waitForElasticsearch(RestHighLevelClient client, long pollingTime)
                             // measure time if health request is successful
                             final Date endTime = new Date();
                             long pollingTimeMeasure = endTime.getTime() - startTime.getTime();
-                            logger.info("Took "+ pollingTimeMeasure + " milliseconds for polling Elasticsearch");
-
+                            logger.info("Took " + pollingTimeMeasure + " milliseconds for polling Elasticsearch");
                             // wake up the waiting thread
                             cdl.countDown();
                         }
-
+                    } catch (ElasticsearchException | SSLException e) {
```

Comment:
Can you try just throwing these and see if it works?

And if not, I can tell you a few ways to handle this properly. For example, the holder can say whether it's a fatal error or not.